lib/src/cli/arguments_validation.dart

```diff
@@ -11,6 +13,10 @@ void validateArguments(ArgResults arguments) {
   checkRootFolderExistAndDirectory(arguments);
   checkPathsToAnalyzeNotEmpty(arguments);
   checkPathsExistAndDirectories(arguments);
+
+  checkObsoletedArguments(arguments, deprecatedOptions).forEach((warning) {
```

Comment:
I'd suggest to use one term (deprecated) instead of two (obsolete and deprecated).

Reply:
fixed